android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MatchingEngineHelper.java

```diff
@@ -332,7 +335,7 @@ private boolean findCloudlet(Location location, Activity ctx, String host, int p
     private boolean verifyLocation(Location location, Activity ctx, String host, int port, String carrierName) throws InterruptedException, IOException, ExecutionException {
         // Location Verification (Blocking, or use verifyLocationFuture):
         AppClient.VerifyLocationRequest verifyRequest =
-                mMatchingEngine.createVerifyLocationRequest(ctx, carrierName, location);
+                mMatchingEngine.createVerifyLocationRequest(ctx, carrierName, location, 0, null);

```

Comment:
I'll check if the new SDK is uploaded, there's new code for this:

    public VerifyLocationRequest.Builder createDefaultVerifyLocationRequest(Context context,
                                                             android.location.Location location) {